scripts/variantstore/wdl/GvsQuickstartIntegration.wdl

```diff
@@ -4,14 +4,43 @@ import "GvsQuickstartVcfIntegration.wdl" as QuickstartVcfIntegration
 import "GvsQuickstartHailIntegration.wdl" as QuickstartHailIntegration
 import "GvsUtils.wdl" as Utils
 
+
+task FilterIntervalListChromosomes {
```

Comment:
nit: you say it below, but a comment here about what you are doing would be nice